x-pack/plugins/features/server/feature_schema.ts

```diff
@@ -33,6 +33,10 @@ const privilegeSchema = Joi.object({
   catalogue: catalogueSchema,
   api: Joi.array().items(Joi.string()),
   app: Joi.array().items(Joi.string()),
+  alerting: Joi.object({
+    all: Joi.array().items(Joi.string()),
+    read: Joi.array().items(Joi.string()),
+  }),
```

Comment:
@restrry do you know if we have an open issue to migrate this to `config-schema`?